optuna/logging.py

```diff
@@ -1,26 +1,24 @@
 import colorlog
 import logging
-from logging import CRITICAL  # NOQA
-from logging import DEBUG  # NOQA
-from logging import ERROR  # NOQA
-from logging import FATAL  # NOQA
-from logging import FATAL  # NOQA
-from logging import INFO  # NOQA
-from logging import WARN  # NOQA
-from logging import WARNING  # NOQA
+from logging import CRITICAL # NOQA
+from logging import DEBUG # NOQA
+from logging import ERROR # NOQA
+from logging import FATAL # NOQA
+from logging import INFO # NOQA
+from logging import WARN # NOQA
+from logging import WARNING # NOQA
 import threading
 
 from optuna import type_checking
 
 if type_checking.TYPE_CHECKING:
-    from typing import Optional  # NOQA
+    from typing import Optional # NOQA
 
 _lock = threading.Lock()
 _default_handler = None  # type: Optional[logging.Handler]
 
 
-def create_default_formatter():
-    # type: () -> colorlog.ColoredFormatter
+def create_default_formatter() -> 'colorlog.ColoredFormatter':
```

Comment:
```suggestion
def create_default_formatter() -> colorlog.ColoredFormatter:
```

We don't have to use forward reference since `colorlog` is imported in runtime.